app/mailers/application_mailer.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationMailer < ActionMailer::Base
+  self.delivery_job = ActionMailer::MailDeliveryJob
```

Comment:
Should this not be `self.delivery_job = MailDeliveryJob` i.e. referencing our custom job class that has the discard_on configuration? Otherwise I don't think anything has changed.

Reply:
Yes you're quite right.... I'll fix it.